src/bp-core/deprecated/15.0.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Deprecated functions.
+ *
+ * @package BuddyPress
+ * @deprecated 15.0.0
+ */
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Output whether signup is allowed.
+ *
+ * @since 1.1.0
+ * @deprecated 15.0.0
+ *
+ * @todo Remove this function. Echoing a bool is pointless.
```

Comment:
You are already deprecating it. So, no todo left.
```suggestion
 * @deprecated 15.0.0
```

Reply:
I believe this will be resloved by https://github.com/buddypress/buddypress/pull/397

Reply:
Yes!